index.bs

```diff
@@ -4676,7 +4688,11 @@ To <dfn>queue reports for delivery</dfn> given a [=set=] of
          forms of interruption, including shutdown, the user agent may need to
          persist reports that are in the process of being sent in some other
          storage.
-    1. [=list/Append=] |report| to |reportsToSend|.
+    1. Let |reportLifeTime| be the lesser of |report|'s
+        [=attribution report/source time=] and [=attribution report/trigger time=].
```

Comment:
It might be clearer to give this a name like `startTime`, as `lifeTime` sounds like a duration, not a time.